test/Makefile

```
@@ -30,7 +30,7 @@ clean:
 	[ "$(TEST_HARNESS_PATH)" == "/" ] || rm -rf $(TEST_HARNESS_PATH)
 
 ## Run all tests
-all: module examples/complete
+all: module examples/complete -run TestExamplesCompleteWithReplication
```

Going to revert this, it's not proper Makefile syntax and I think it was added by mistake.

```suggestion
all: module examples/complete
```